x-pack/platform/plugins/shared/streams/common/constants.ts

```diff
@@ -6,3 +6,10 @@
  */
 
 export const ASSET_VERSION = 1;
+
+export const STREAMS_FEATURE_PRIVILEGES = {
+  assets: 'manage_assets',
+  read: 'read',
+  write: 'manage_streams',
+  show: 'show',
+} as const;
```

Comment:
Should we differentiate UI vs API privileges? 
I think API privileges require to be unique across all plugins' API privileges, so probably need to use read_stream, manage_streams and manage_assets.

Do we need the distinction between manage_streams and manage_assets for the APIs?

I would think this is enough:
```
all: { api: ["read_stream", "manage_stream"], ui: ["show", "manage"] }
read: { api: ["read_stream"], ui: ["show"] }
```